Update docs after beta4 (#23)
sylabs/singularity-cri-userdocs: docs/constraints.rst

Using image from private registry
---------------------------------

Unfortunately, it is not possible to use on-prem Sylabs library or pull private SIF images when working with

> Singularity-CRI. This is a subject to change soon, we are working with Kubernetes maintainers on

Reviewer: ```suggestion
Singularity-CRI. However, this is expected to change soon as we work with Kubernetes maintainers on
```
